pipelines/bioasq/dvc.yaml

```diff
@@ -19,7 +19,7 @@ stages:
         approach: "mesh-xlinear"
         tagged_data_path: "../../data/processed/bioasq/tagged_data_${test_number}_xlinear.json"
         threshold: 0.2
-      - model_path: "../../models/bertmesh/model.pt"
```

Comment:
Not pt anymore?

Reply:
We are using `PretrainedModel` now which saves a `bin` which is the same as `pt` actually